src/sagas.js

```diff
@@ -48,7 +69,7 @@ export function* rootSaga() {
     spinnerInit(),
     loginInit(),
     logoutButtonInit(),
-    notesInit(),
+    // notesInit(),
```

Comment:
Why remove `notesInit()` ?

Reply:
```
export function* notesInit() {
  yield all([
    readHelpSeenState(),
    takeEvery(CHANGE_USER_HELP_SEEN_ACTION, saveHelpSeenState),
  ]);
}
```

It's better to use help on demand and just make intuitive UI

Reply:
How is `notesInit()` related to help?